Nodejs/Product/Nodejs/TypingsAcquisition.cs

```diff
@@ -28,22 +28,47 @@
 
 namespace Microsoft.NodejsTools {
     internal class TypingsAcquisition {
-        private const string Tsd = "tsd";
-        private const string TsdExe = Tsd + ".cmd";
+        private const string TypingsAcquisitionTool = "typings";
+        private const string TypingsAcquisitionToolExe = TypingsAcquisitionTool + ".cmd";
```

Comment:
nit: to keep things consistent... TypingsTool to refer to typings itself, TypingsAcquisition to refer to the automatic feature?